### DISCUSSION TITLE
problem with Splitting mesh file

### DISCUSSION THREAD

**Discussion by dewan1988** [idaholab/moose#26771] [Q&A General]
Dear MOOSE Experts
I have splited my mesh file into 128 chunks and trying to run my input file but its showing the following error. How can I solve this problem?
*** ERROR ***
The following error occurred in the object "mixing_len_aux_ker", of type "WallDistanceMixingLengthAux".

WallDistanceMixingLengthAux only supports replicated meshes

**Comment by GiudGiud** [comment]
Hello

So either stop using distributed meshes. 

OR

You ll need to use the WallDistanceMixingLengthAux in a simple simulation with no flow and no distributed mesh.
Save the output in a checkpoint file preferrably or an exodus file
In your real distributed simulation, do not use the  WallDistanceMixingLengthAux, instead load the output field from the checkpoint/exodus file
